src/design-system-team.md.njk

```diff
@@ -19,7 +19,7 @@ If you want to contact the team you can
 - Hanna Laakso – Frontend Developer
 - Joe Lanman – Lead Interaction Designer
 - Kelly Lee – Delivery Manager
-- Mark Green – Lead Technical Writer
+- Eoin Shaughnessy – Technical Writer
```

Comment:
@EoinShaughnessy Sorry for the nitpick! But would you mind moving your name up so the list is in alphabetical order?

Reply:
I've fixed this as I paired on this with Eoin and didn't spot the order.